lib/galaxy/tools/unzip_collection.xml

```diff
@@ -1,29 +1,39 @@
 <tool id="__UNZIP_COLLECTION__"
-      name="Unzip Collection"
+      name="Unzip"
       version="1.0.0"
       tool_type="unzip_collection">
+  <description>collection into multiple ones</description>
   <type class="UnzipCollectionTool" module="galaxy.tools" />
   <action module="galaxy.tools.actions.model_operations"
           class="ModelOperationToolAction"/>
     <edam_operations>
         <edam_operation>operation_3359</edam_operation>
     </edam_operations>
   <inputs>
-    <param type="data_collection" collection_type="paired" name="input" label="Input Paired Dataset" />
+    <param type="data_collection" collection_type="paired" name="input" label="Paired List Collection to unzip" />
```

Comment:
Like in the other tool suggestion - I think this should be more generic since this natively takes in pairs and not list of pairs - I think if we changed it to "Paired input" we can circumvent that argument?

Reply:
Changed to `Paired input to unzip`